linux/Jenkinsfile

```diff
@@ -193,6 +193,9 @@ def jenkinsStepDeb() {
 // function handle both Alpine, RedHat and Suse as DISTRO
 def jenkinsStepNonDeb(String DISTRO) {
     echo "Installer Job for Temurin jdk ${VERSION} - ${ARCH} - ${DISTRO}"
+    if (("${DISTRO}" == 'RedHat' || "${DISTRO}" == 'Suse') &&  ("${ARCH}" == 'armv7l')) {
```

Comment:
The brackets around the ARCH close seem superfluous, but it won't do any harm ...